client/src/containers/FileViewPage/index.jsx

```diff
@@ -45,6 +46,14 @@ class FileViewPage extends React.Component {
     navigator.clipboard.writeText(this.filepath);
   }
 
+  handleOpenRaw() {
+    const { match } = this.props;
+    const { username, reponame, branch } = match.params;
+
+    window.location
+      .replace(`${process.env.REACT_APP_RAW_SERVER_URL}/${username}/${reponame}/${branch}/${this.filepath}?token=${localStorage.getItem('token')}`);
```

Comment:
Maybe it's better to reference process.env in app.config.js like that: const rawServerUrl = process.env.REACT_APP_RAW_SERVER_URL;
and then to import rawServerUrl in FileViewPage ?